pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -211,7 +214,12 @@ void ATA::CreateHDDinfo(u64 sizeSectors)
 	 * bit 14: NOP
 	 * bit 15: (Obsolete)
 	 */
-	WriteUInt16(identifyData, &index, static_cast<u16>((1 << 14) | (1 << 5) | /*(1 << 1) | (1 << 10) |*/ 1)); //word 82
+	// clang-format off
+	WriteUInt16(identifyData, &index, (		//word 82
+		(1 << 0) |							//implemented
+		(1 << 5) |							//implemented
+		(1 << 14)							//Fixed
```

Comment:
```suggestion
		(1 << 14)							//implemented
```

The NOP command is implemented